pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -647,7 +648,10 @@ func (p *Provisioner) setAdminOpsAPIClient() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to build object store admin ops API connection")
 	}
-
+	rookLogLevel, _ := k8sutil.GetOperatorSetting(p.context.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_LOG_LEVEL", "INFO")
```

Comment:
We can just check like so instead of fetching the cm

`if logger.LevelAt(capnslog.DEBUG) {`